jinja2cli/cli.py

```diff
@@ -197,6 +197,12 @@ def cli(opts, args):
             ext = 'jinja2.ext.' + ext
         extensions.append(ext)
 
+    # Use only a specific section if needed
+    if opts.section:
+        section = opts.section
+        if section in data:
```

Comment:
> imo, I'd want to throw the KeyError here.

I don't think that throwing a exception is user friendly. Would a message like `ERROR: unknown section. Exiting` and existing with a non 0 status code be OK for you?